src/GitLink/Program.cs

```diff
@@ -8,6 +8,9 @@
 namespace GitLink
 {
     using System;
+    using System.CommandLine;
```

Comment:
Our regular approach is not to use prerelease packages (since it will not allow you to create stable versions for NuGet). But since this is a tool and not a library, it should be fine.